### PR TITLE
Sol draft order changes

### PR DESCRIPTION
**Pull request by shashisk97**
### JIRA link (if applicable) ###
[PRL-2451](https://tools.hmcts.net/jira/browse/PRL-2451)

### Change description ###
Solicitor draft an order

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: definitions/private-law/json/CaseEvent.json [hunk 1]
```diff
@@ -1248,6 +1248,52 @@
...
   },
+  {
+    "LiveFrom": "31/03/2022",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "draftAnOrder",
+    "Name": "Draft an order",
+    "Description": "Draft an order",
+    "DisplayOrder": 161,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "ALL_FINAL_ORDERS_ISSUED(doesOrderClosesCase=\"Yes\"):1;*",
```
Comment (AlokDatta) [inline definitions/private-law/json/CaseEvent.json:1259]: not sure on the post condition. also precondition, guessing started when the case is submitted.

File: definitions/private-law/json/CaseEvent.json [hunk 2]
```diff
@@ -1248,6 +1248,52 @@
...
+    "LiveFrom": "26/07/2022",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "editAndApproveAnOrder",
+    "Name": "Edit and approve a draft order",
+    "Description": "Edit and approve a draft order",
+    "DisplayOrder": 27,
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/populate-draft-order-dropdown",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/judge-or-admin-edit-approve/about-to-submit",
+    "PreConditionState(s)": "SUBMITTED_PAID",
+    "PostConditionState": "*",
```
Comment (AlokDatta) [inline definitions/private-law/json/CaseEvent.json:1276]: Precondition does not match with above event precondition

File: definitions/private-law/json/CaseEvent.json [hunk 3]
```diff
@@ -1248,6 +1248,52 @@
...
+    "LiveFrom": "26/07/2022",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "adminEditAndApproveAnOrder",
+    "Name": "Edit and approve a draft order",
+    "Description": "Edit and approve a draft order",
+    "DisplayOrder": 28,
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/populate-draft-order-dropdown",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/judge-or-admin-edit-approve/about-to-submit",
+    "PreConditionState(s)": "SUBMITTED_PAID",
+    "PostConditionState": "*",
```
Comment (AlokDatta) [inline definitions/private-law/json/CaseEvent.json:1292]: Precondition does not match with above event precondition

**Comment by shashisk97** [comment]
Not needed

**Pull request closed without merging**